File: Java/MicroserviceApp/route-service/src/main/java/com/example/route/domain/City.java
package com.example.route.domain;

import jakarta.persistence.*;

@Entity
@Table(name = "cities")
public class City {
    @Id
    @Column(name = "id", nullable = false, updatable = false, length = 64)
    private String id;
    @Column(nullable = false, unique = true, length = 120)
    private String name;
    private String state;
    private Integer population;

    public City() {}
    public String getId() { return id; }
    public String getName() { return name; }
    public String getState() { return state; }
    public Integer getPopulation() { return population; }
}

File: NET/MicroserviceApp/RouteService/Database/Model/City.cs
using System.ComponentModel.DataAnnotations;
using System.ComponentModel.DataAnnotations.Schema;

namespace RouteService.Database.Model
{
    public class City
    {
        [Key]
        [Column("id")]
        public string Id { get; set; }

        [Column("name")]
        public string Name { get; set; }

        [Column("state")]
        public string State { get; set; }

        [Column("population")]
        public int Population { get; set; }
    }
}
